activerecord/lib/active_record/transactions.rb

```diff
@@ -206,6 +206,19 @@ def transaction(options = {}, &block)
         connection.transaction(options, &block)
       end
 
+      def start_sandbox
```

Comment:
should these be marked private, nodoc, and/or named `_start_sandbox`? Otherwise it's public API and needs docs.